README.md

```diff
@@ -50,7 +50,7 @@ For those who want to break the norms. You can create custom looks in lualine.
 
 ## Performance compared to other plugins
 
-Unlike other statusline plugins lualine loads only defined components, nothing else.
+Unlike other statusline plugins, lualine loads only a defined set of components, nothing else.
```

Comment:
```suggestion
Unlike other statusline plugins, lualine loads only user defined set of components, nothing else.
```

I feel like a set doesn't convey the full message . It's suppose to mean that lualine only loads components that user used in their config and leaves out the others.

Reply:
You're right, I've updated it.